apidoc/Titanium/Media/Media.yml

```diff
@@ -2054,7 +2054,9 @@ summary: |
 properties:
   - name: success
     summary: Function to call when the photo gallery is closed after a successful selection.
-    type: Callback<CameraMediaItemType>
+    description: |
+        If <Titanium.Media.allowMultiple> is `true` or <Titanium.Media.selectionLimit> is set, Callback<CameraMediaMultipleItemsType> is called otherwise Callback<CameraMediaItemType>.
```

Comment:
This sentence is a bit awkward. Let's change it to this...
```
If <Titanium.Media.allowMultiple> is `true`, then Callback<CameraMediaMultipleItemsType> will be invoked. Otherwise Callback<CameraMediaItemType> will be invoked for single selection.
```